script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_2_2.txt

```diff
@@ -11,7 +11,7 @@ No, rather than Arcueid, this is about me.
 If I stay here――― If I become involved with Arcueid, I'm bound to run into him.
 <Page4>
 That guy is crazy.
-Whether it's him being a vampire or his flames, more than that, his eyes are beyond normal.
+Whether it's him being a vampire or his flames, more than that, his eyes glitter with madness.//to be rechecked
```

Comment:
Even more than him being a vampire or his flames, his eyes are full of madness.

Reply:
I like that, but maybe "the flames"